app_listenAndServe.go

```diff
@@ -22,7 +24,11 @@ func (app *App) ListenAndServe(addr ...string) error {
 	if len(addr) > 0 {
 		bind = addr[0]
 	} else {
-		if !app.flagsRegistered {
+		bind = os.Getenv("PORT")
+		if len(bind) > 0 && !strings.Contains(bind, ":") {
```

Comment:
strings.HasPrefix looks more good

Reply:
nope, you can use `127.0.0.1:9090` hack if we use `Contains`. `PORT` just for compatibility with Heroku and others